### ISSUE TITLE
Create a profile for each controller in Playnite Fullscreen

### ISSUE THREAD

**Issue by GreenRabbits** [JosefNemec/Playnite#1752]
It would be great to have a profile assigned to each controller connected when Playnite Fulllscreen is running. Each profile would have its theme, its filters and optionally its library.

**Comment by JosefNemec** [comment]
What do you mean by controller? Like physical gamepad?

**Comment by GreenRabbits** [comment]
> What do you mean by controller? Like physical gamepad?
Yes, I mean a physical gamepad. For those who have two controllers or more, it could be useful to assign a profile to each controller.
For example, a girl named Emma has her own gamepad. When she launches playnite fullscreen mode with her controller connected, it displays a pink theme, with her favorite games.
But, her brother Noah has also his own gamepad. And, when he starts playnite, the theme and the games change.
When both gamepads are connected during playnite launching, a pop-up asks how wants to control playnite, and have his own UI.

**Comment by JosefNemec** [comment]
We currently only support XInput devices and there's really no way how to tell which controller was connected (they are just generic XInput devices) without reworking the whole input system.

I also think that the use case you propose is very niche, so don't expect this to be added in near future.

Also linking #684 to this.

**Comment by GreenRabbits** [comment]
> We currently only support XInput devices and there's really no way how to tell which controller was connected (they are just generic XInput devices) without reworking the whole input system.
> 
> I also think that the use case you propose is very niche, so don't expect this to be added in near future.
> 
> Also linking #684 to this.

Ok, no problem. I thought this enhancement would requiere too much work and changes. I think that the answer is no, but, is it possible to make Playnite Fullscreen controllable by only one gamepad ?

**Comment by JosefNemec** [comment]
That shouldn't be that hard to add. Please open new issue for it and I'll add option where only first controller's input is accepted.

**Comment by JosefNemec** [comment]
> I also think that the use case you propose is very niche, so don't expect this to be added in near future.

Closing based on this comment. Option to only accept primary controller has been added already in later releases.
